src/applications/gi-sandbox/reducers/search.js

```diff
@@ -166,17 +175,52 @@ export default function(state = INITIAL_STATE, action) {
         query: { ...state.query, ...action.payload },
         geocodeInProgress: true,
       };
+    case GEOCODE_FAILED:
+      return {
+        ...state,
+        error: true,
+        geocodeError: action.code,
+        geocodeInProgress: false,
+        geolocationInProgress: false,
```

Comment:
which of these in progresses are being used?
please remove the one that is not being used by the app